lib/utils/render-note-to-html.ts

```diff
@@ -17,6 +17,9 @@ export const renderNoteToHtml = (content: string) => {
       markdownConverter.setOption('simpleLineBreaks', false); // override GFM
       markdownConverter.setOption('ghMentions', false);
       markdownConverter.setOption('smoothLivePreview', true);
+      markdownConverter.setOption('tables', true); // table syntax
+      markdownConverter.setOption('strikethrough', true); // ~~strikethrough~~
+      // markdownConverter.setOption('emoji', true); // emoji support like :smile:
```

Comment:
I'm assuming this isn't supposed to be here?

it's not something I would want to support just because of how inconsistent and undocumented it would seem (at least at this time).

Reply:
> I also added the emoji module here, but in a comment, because I want to remind myself that it exists if we ever want to support those!

But okay OKAY fine I'll take it out and make this note to myself somewhere else

Reply:
oh I missed that in the description, sorry!

it's fine to leave it in but maybe instead explain that _in the comment_

```ts
// markdownConverter.setOption('emoji', true); // earmarked to remind us that this exists in case we decide to support it in the future
```

probably we don't need to earmark it because if we want to add emoji it won't be hard to find, but I don't mind commented out code if there's an explained reason for it